docs/Migrations.md

```diff
@@ -0,0 +1,22 @@
+# Diesel Migrations
+
+## Pre-requisites
+
+* `cargo install diesel_cli --no-default-features --features postgres`
+
+## Generating new migrations
+
+* `cd components/[builder-originsrv, builder-jobsrv, builder-sessionsrv]`
+* `diesel migration generate <your migration name>`
```

Comment:
If there's a naming convention for these, it'd be nice to link to it or somehow cite an example or two, since it might not be the easiest thing to change once it's executed.

Reply:
Does Postgres need to be running for this to work, or are we just generating files?

Reply:
After installing, when I run this, I get the following:

```
root@vagrant:/src/components/builder-originsrv# diesel migration generate add_get_origin_integration
Unable to find migrations directory in this directory or any parent directories.
```